app/models/price_range.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class PriceRange < ApplicationRecord
```

Comment:
👏 bardzo sprytnie. Zastanawiam sie tylko nad balansem zyskow i strat tego rozwiazania. Dodatkowy model i tyle szycia dla zwyklych filtrow. Moze jednak dalo sie to prosciej zrobic? :)